modules/heat_conduction/src/materials/SemiconductorLinearConductivity.C

```diff
@@ -36,7 +36,7 @@ SemiconductorLinearConductivity::SemiconductorLinearConductivity(const InputPara
     _base_name(isParamValid("base_name") ? getParam<std::string>("base_name") + "_" : ""),
     _electric_conductivity(declareProperty<Real>(_base_name + "electrical_conductivity")),
     _delectric_conductivity_dT(declarePropertyDerivative<Real>(
-        _base_name + "electrical_conductivity", getVar("temp", 0)->name()))
+        _base_name + "electrical_conductivity", coupledName("temp", 0)))
```

Comment:
same here

Reply:
and many other places

Reply:
changed everywhere. Causes Marmot to break on one file. 
Can you make the patch or should I revert on file here?

Reply:
*everywhere
Only on the declare*Derivatives. The get*Derivative should be fine querying dCd47.7